eclipse-plugin/sapl-eclipse-ui/src/main/java/io/sapl/grammar/ui/quickfix/SAPLQuickfixProvider.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2017-2024 Dominic Heutelbeck (dominic@heutelbeck.com)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+ * SAPL generated by Xtext
+ */
+package io.sapl.grammar.ui.quickfix;
+
+import org.eclipse.xtext.ui.editor.quickfix.DefaultQuickfixProvider;
+
+/**
+ * Custom quickfixes.
+ *
+ * See
+ * https://www.eclipse.org/Xtext/documentation/310_eclipse_support.html#quick-fixes
+ */
+public class SAPLQuickfixProvider extends DefaultQuickfixProvider {
+
+    // @Fix(SAPLValidator.INVALID_NAME)
```

Comment:
Same here. 

Proposal for implementation: Lazy boolean operators to be replaced by eager ones where applicable. I can explain in detail. DM me.

Reply:
Removed the example code for now. I will contact you regarding your proposal after fixing the remaining issues.